test/class-test.js

```diff
@@ -1,16 +1,29 @@
-import { promises as fs } from 'fs';
+import express from 'express';
+import Server from '../index.js';
+import { config } from '../config.js';
 import got from 'got';
 import assert from 'assert';
-import {} from '../index.js';
+import { promises as fs } from 'fs';
 
 describe('server', () => {
+    let server = null;
+
+    before(async () => {
+        server = new Server(config.port, express());
+        await server.start();
+
+    });
+
+    after(async () => {
+        await server.finish();
+    });
+
```

Comment:
let's give our test more meaningful name, e.g. `Server should respond with the correct content and status`